### ISSUE TITLE
No such method exception even with latest version of forge

### ISSUE THREAD

**Issue by Prolepsis** [BluSunrize/ImmersiveEngineering#1069]
Hi,

I've got the classic "no such method error" while trying to integrate your mod into Glixyl's Entropy modpack. I've done a fair bit of research into potential fixes (listed below) but the issue persists. My error log is here: http://pastebin.com/zaxrMkfi

List of stuff I've tried already:
---
1. I've checked similar errors in your issues section and it seems the standard answer is update forge. So I've gone through and updated all the forge mods I could get my hands on:

ForgeMultipart-1.7.10-1.2.0.347 (universal) - released 04/04/2016

2. Multipart was the only forge mod listed in the /mods/ directory for this modpack. So I also downloaded and added the most recent recommended release of forge sans multipart (not sure if it's necessary, but just in case) 

forge-1.7.10-10.13.4.1558-1.7.10 (universal) 

3. In reading your guidelines, I noted that fastcraft can cause issues. I upgraded to the most recent version (1.23) and it crashed as well. I also removed fastcraft entirely and the same thing happened.

----

Any other things I should try?

Thanks so much!

**Comment by mallrat208** [comment]
It's worth noting that your log there indicates you're using Forge 10.13.2.1342  not 10.13.4.1558

**Comment by Prolepsis** [comment]
Thanks so much for responding.

Hmm, that's strange. I can see it in the logs as well. yet the 1558 version is present in the ./mods/1.7.10 folder. In fact, there is no jar listed in the /mods folder with 1342 in it at all. The only reference left to 1342 comes from the /cache/ folder, rather than mods. Could it be that th e

I deleted the contents of the folder and tried again, no no avail and no change in the source of the error. In digging a little further, I found the older forge version renamed as ./bin/modpack.jar. 

So what I did was rename the forge download as "modpack.jar" and replaced the existing one.
